src/UXClient/Interfaces/ChartComponent.ts

```diff
@@ -182,7 +182,13 @@ class ChartComponent extends Component {
             subtitle.append('h4')
                 .text(d.splitBy)
                 .attr('class', 'tsi-tooltipSeriesName');
-        }
+		}
+		
+        if (cDO.variableAlias){
```

Comment:
title components should be placed in their own div that uses a bottom margin to separate from tooltipattributes, groupname, seriesname, and variable alias should have classes to differentiate them, and series name should be display: none if it is empty string